### PR TITLE
Implemented chrome headless

### PR DESCRIPTION
**Pull request by andkochergin**

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -1,10 +1,19 @@
...
 end
+
+Capybara.register_driver :headless_chrome do |app|
+  opts = Selenium::WebDriver::Chrome::Options.new
+  chrome_args = %w[--headless --no-sandbox --disable-gpu --window-size=1920,1080]
+  chrome_args.each { |arg| opts.add_argument(arg) }
+  Capybara::Selenium::Driver.new(app, browser: :chrome, options: opts)
+end
+
+Capybara.javascript_driver = :headless_chrome
```
Comment (yahorzhylinski) [inline spec/spec_helper.rb]: Make test run either headless or real browser by env variable and by default with driver

Let's say
If you run tests like `rspec spec` browser should be visible.
If you run `HEADLESS=true rspec spec` tests are in headless mode.

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
-Capybara.current_driver = :selenium_chrome
 Capybara.app_host = 'https://onliner.by'
 
 RSpec.configure do |config|
   config.include Capybara::DSL
 end
+
+Capybara.register_driver :headless_chrome do |app|
+  opts = Selenium::WebDriver::Chrome::Options.new
+  chrome_args = %w[--headless --no-sandbox --disable-gpu --window-size=1920,1080]
```
Comment (yahorzhylinski) [inline spec/spec_helper.rb]: add this arg: `--disable-dev-shm-usage`

Comment (yahorzhylinski) [reply spec/spec_helper.rb]: Please read about this flag

File: Gemfile [hunk 3]
```diff
@@ -6,6 +6,6 @@ gem 'capybara'
 gem 'overcommit'
 gem 'rspec'
 gem 'rubocop', require: false
-gem 'webdrivers'
-
+gem 'webdrivers', '~> 3.7', '>= 3.7.2'
```
Comment (yahorzhylinski) [inline Gemfile]: Have you checked all tests wuth headless? Are they all passed?

File: spec/spec_helper.rb (hunk 1 above)
Comment (andkochergin) [reply spec/spec_helper.rb]: done

File: spec/spec_helper.rb (hunk 2 above)
Comment (andkochergin) [reply spec/spec_helper.rb]: done

File: Gemfile (hunk 3 above)
Comment (andkochergin) [reply Gemfile]: Yes, all tests passed

**Pull request merged**